tests/test_config/test_config.py

```diff
@@ -226,3 +227,23 @@ def test_config_parse_error_has_no_snippet() -> None:
     snippet = reporter.messages[config][0].snippet
 
     assert snippet == ""
+
+
+def test_allow_pylint_comments() -> None:
+    """Test that checks whether the allow-pylint-comments configuration option works as expected when it is
+    set to True"""
```

Comment:
As above, make sure the ending `"""` is on a new line. Same with the second test case below.